docs/migrating.md

```diff
@@ -0,0 +1,10 @@
+# Migrating rules from rules_python to rules_py
+
+rules_py tries to closely mirror the API of rules_python, so a migration is a "drop-in replacement" for the majority of use cases.
+
+## Replace load statements
+
+Instead of loading from `@rules_python//python:defs.bzl`, you load from `@aspect_rules_py//py:defs.bzl`.
```

Comment:
Same rule as "we", remove "you" for technical writing.

```suggestion
Instead of loading from `@rules_python//python:defs.bzl`, load from `@aspect_rules_py//py:defs.bzl`.
```